docker/redmine/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM centos:7
+MAINTAINER Eric Helms
+
+RUN yum -y install vim ruby ruby-devel gcc make libxml2-devel libxslt-devel postgresql-devel libvirt-devel wget git mysql-devel ImageMagick-devel
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+RUN \curl -sSL https://get.rvm.io | bash -s stable
+RUN /bin/bash -l -c "rvm install 1.9.3"
+RUN /bin/bash -l -c "rvm --default use 1.9.3"
+RUN /bin/bash -l -c "gem install bundler --no-ri --no-rdoc"
+
+RUN git clone https://github.com/theforeman/redmine.git
```

Comment:
The trick is this line (https://github.com/Katello/katello-deploy/pull/185/files#diff-200c6f3d26920a58240b6405a851dba0R18) where I am attempting to do a bundle install during build; a sort of optimization. Otherwise bundle occurs everytime you bring the container up. Granted, I took this from some Foreman container work where bundle took an annoying amount of time when you wanted to just snap open a new container so I could remove this part of it. When you mount a local copy of Redmine through the volumes, it does overwrite this.